### ISSUE TITLE
Use of DEM in the Noise_level_from_Traffic block.

### ISSUE THREAD

**Issue by SantiagoRVR** [Universite-Gustave-Eiffel/NoiseModelling#669]
**NoiseModelling version**
NoiseModelling v4.0

**Question**
I want to obtain a Noise Map using only traffic data as the source of noise. For this, I am using the Noise_level_from_traffic block. I have 5 different input layers from which 3 are mandatory to use the block. My layers are:

  - Buildings: Containing the footprint of the building, a column for the PK and the height for each building.
  - Receivers: Geojson file with a Z value of 2
  - Roads: Containing the traffic per segment.
  - Ground: Absorption parameter
  - DEM: Elevation map

My question is related to the use of the DEM in the model, as I have both DSM and DTM but not really sure which one to use. For what I understand, the DEM would be used to obtain the lowest point of the map at every location and on top of this DEM, the rest of the layers would be overlaid. This meaning that I have the terrain elevation where a building is located, then the footprint is laid atop and extruded to the height of the building, the final elevation being DEMheight + Buildingheight. The same thing would happen for each of the receivers that would be at an elevation of DEM + 2m, roads at DEM + 0.5m, and with height information, the noise would be evaluated. The specific question is, is this how the model handles the DEM? If it is the case, then I should use the DTM as the DEM. But if this is not how the model works, could you help me explain how the DEM would be used?

**Comment by MaguetteD** [comment]
Hi,
we use DTM as the DEM in NoiseModelling. This approach ensures that all structures and noise sources are accurately placed in relation to bare ground elevation, providing a realistic basis for noise level calculations.
The final noise map will be calculated by taking the DTM as the base elevation and adding the respective heights of buildings, receptors and roads, for buildings: DTM + buildings heights and for receivers: DTM +2m.

You can refer to the documentation below.
https://noisemodelling.readthedocs.io/en/latest/Input_buildings.html

**Comment by SantiagoRVR** [comment]
Thanks for the reply. 
I understand then that the height of the buildings is the DEM (DTM in this case) plus the Height in the column. In the case of the receivers and roads, when uploading a map, I have uploaded a PointZ and LineStringZ files. On these files I have set 2m and 0.05m as their Z value respectively. I assume then that their final height would be DEM + 2m and DEM + 0.05, right? Meaning that for receivers it would not be DEM + 2m I set + 2m set by the model. Am I correct?
